storage/libradb/src/libradb_test.rs

```diff
@@ -456,7 +455,8 @@ proptest! {
         assert!(events.is_empty());
     }
 
-    #[test]
+    /* #[test]
+    // TODO: please fix me!
```

Comment:
does this test serve any purpose? why is it even here? it feels like we really needed a proper place to put end-to-end configuration tests (libra <-> move)

Reply:
Strange test indeed. I can fix it be querying NewEpochEvent instead of DiscoverySetChangeEvent, or remove it all-together. Let me try to fix it...